CrisisTextLine/Elmer
# Use students list from course to establish eligible melters instead o…

#### What's this PR do?
Use students list from course to establish eligible melters instead of just those in the grade log

#### Where should the reviewer start?
autoForceMelt.js

#### How should this be manually tested?
Change the traineeEmail address to your own (I use `assaf+${studentObj.id}@crisistextline.org)
Comment out the sections of checkEnrollmentAndMeltUser that cause melt
Run the script and ensure that the appropriate emails are being sent out.

#### Any background context you want to provide?
#### What are the relevant tickets?
https://admin.crisistextline.org/jira/secure/RapidBoard.jspa?rapidView=12&selectedIssue=AT-261&quickFilter=44

#### Questions:
